### DISCUSSION TITLE
How to run Dbmate without Root?

### DISCUSSION THREAD

**Discussion by zer3fD** [purton-tech/rust-on-nails#49] [Q&A] [answered]
Hello,
I have setup a new Dev container of Rust On Nails from the VSCode(not from local directory and pulling git).
When i run: `dbmate new user_tables`
The Error: Error: unable to create directory `/workspace/crates/db/migrations`

But when i run sudo it works,But unable to write changes in VsCode as it get Permission Denied Error.

How do i run dbmate locally?

**Comment by Fodesu** [comment]
I encountered the same problem as you. Have you found a solution?

**Reply by Fodesu** [reply, marked as answer]
find solution:
 add a  `privileged : true` in docker-compose.yml and rebuild the docker image
